.github/workflows/reusable_dockerfile_pipeline.yml

```diff
@@ -142,6 +142,7 @@ jobs:
           OUTPUT_SHORT_SHA: ${{ needs.prepare-env.outputs.output_short_sha }}
           OUTPUT_IMAGE_NAME: ${{ needs.prepare-env.outputs.output_image_name }}
         with:
+          context: .
```

Comment:
does this need to be customizable? If so we should add it a non-required input with the default `.`

Reply:
hello @sysrex 
as matt said, if we need to add it, it would be great to add another input with the default value `.` to make it more flexible with any pipeline that we need to use, you can use as a reference something like: [link](https://github.com/celestiaorg/.github/blob/main/.github/workflows/reusable_dockerfile_pipeline.yml#L6-L10)